sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet/gluoncv_ssd_mobilenet_neo.ipynb

```diff
@@ -4,18 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>Change</p><p>While using the Pascal VOC <strong>dateset,</strong> please be aware of the database usage rights. The VOC data includes images obtained from flickr's website. Use of these images must respect the corresponding terms of use: <a href="https://www.flickr.com/help/terms" target="_blank">https://www.flickr.com/help/terms</a></p><p>To</p><p>While using the Pascal VOC <strong>dataset</strong>, please be aware of the database usage rights. The VOC data includes images obtained from flickr's website. Use of these images must respect the corresponding terms of use: <a href="https://www.flickr.com/help/terms" target="_blank">https://www.flickr.com/help/terms</a></p><p> </p><p> </p><p>Change</p><p>Default EBS Volume size for SageMaker Notebook instances is 5GB. While performing this step if you run out of storage then <strong>conisder</strong> increasing the volume size. One way to do so is by using AWS CLI as documented here.</p><p>To</p><p>Default EBS Volume size for SageMaker Notebook instances is 5GB. While performing this step if you run out of storage then <strong>consider</strong> increasing the volume size. One way to do so is by using AWS CLI as documented here.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
done